src/orders/utils/returns.test.ts

```diff
@@ -0,0 +1,161 @@
+      stocks: [
+        {
+          __typename: "Stock",
+          id: "U3RvY2s6NTEzMg==",
+          quantity: 456,
+          quantityAllocated: 0,
+          warehouse: {
+            __typename: "Warehouse",
+            id: "V2FyZWhvdXNlOjlkYjY4NWQxLWViMTktNDU2ZS05ODMyLTMxODA3ZWM0NDdhOQ==",
+            name: "Europe",
+          },
+        },
+        {
+          __typename: "Stock",
+          id: "U3RvY2s6NTEzNA==",
+          quantity: 414,
+          quantityAllocated: 44,
+          warehouse: {
+            __typename: "Warehouse",
+            id: "V2FyZWhvdXNlOjdmZDA0OGI0LWYwNzItNDZmMi1iMDMyLTc3ZWU3MDNiMzM3Yg==",
+            name: "Americas",
+          },
+        },
+      ],
+      product: {
+        __typename: "Product",
+        id: "UHJvZHVjdDo3Ng==",
+        isAvailableForPurchase: true,
+      },
+    },
+    productName: "Coconut Juice transaction flow",
+    productSku: "3944859",
+    quantity: 1,
+    quantityFulfilled: 2,
+    quantityToFulfill: 0,
+    totalPrice: {
+      __typename: "TaxedMoney",
+      net: {
+        __typename: "Money",
+        amount: 21,
+        currency: "EUR",
+      },
+      gross: {
+        __typename: "Money",
+        amount: 21,
+        currency: "EUR",
+      },
+    },
+    unitDiscount: {
+      __typename: "Money",
+      amount: 10.5,
+      currency: "EUR",
+    },
+    unitDiscountValue: 10.5,
+    unitDiscountReason:
+      "Promotion: UHJvbW90aW9uOjYyNTgyZTZmLWNjYzQtNDMyNC05MmM2LWI3ZDg5ZjFjZWJmMw==",
+    unitDiscountType: DiscountValueTypeEnum.FIXED,
+    undiscountedUnitPrice: {
+      __typename: "TaxedMoney",
+      currency: "EUR",
+      gross: {
+        __typename: "Money",
+        amount: 21,
+        currency: "EUR",
+      },
+      net: {
+        __typename: "Money",
+        amount: 21,
+        currency: "EUR",
+      },
+    },
+    unitPrice: {
+      __typename: "TaxedMoney",
+      gross: {
+        __typename: "Money",
+        amount: 10.5,
+        currency: "EUR",
+      },
+      net: {
+        __typename: "Money",
+        amount: 10.5,
+        currency: "EUR",
+      },
+    },
+    thumbnail: {
+      __typename: "Image",
+      url: "",
+    },
+    orderLineId:
+      "T3JkZXJMaW5lOmZjNzRlNDliLWQyNTItNDVhYS1hMWU0LWQ2MGU5MzFiNjNlOA==",
+  },
+];
+const mockedItemQuantities: FormsetData<LineItemData, number> = [
+  {
+    data: {
+      isFulfillment: true,
+      isRefunded: false,
+      orderLineId:
+        "T3JkZXJMaW5lOmZjNzRlNDliLWQyNTItNDVhYS1hMWU0LWQ2MGU5MzFiNjNlOA==",
+    },
+    id: "RnVsZmlsbG1lbnRMaW5lOjU1Ng==",
+    label: "",
+    value: 1,
+  },
+  {
+    data: {
+      isFulfillment: true,
+      isRefunded: false,
+      orderLineId:
+        "T3JkZXJMaW5lOmZjNzRlNDliLWQyNTItNDVhYS1hMWU0LWQ2MGU5MzFiNjNlOA==",
+    },
+    id: "RnVsZmlsbG1lbnRMaW5lOjU1Nw==",
+    label: "",
+    value: 0,
+  },
+];
+
+describe("getItemPriceAndQuantity", () => {
+  it("returns undefined unitPrice when ID is not found", () => {
```

Comment:
question: Is it enough to test one case here?

Reply:
This is a regression test, I'm only testing case when dasbhoard crashed